### PR TITLE
Fix the build

### PR DESCRIPTION
**Pull request by rarguello**

### CODE REVIEW

**Comment by wolfc** [comment]
I have rebased this branch on top of fedora16.

File: bootstrap/build.xml [hunk 1]
```diff
@@ -88,6 +89,8 @@
          destdir="output/code-quality">
          <classpath>
             <pathelement path="output/core"/>
+            <pathelement path="output/plugins"/>
```
Comment (wolfc) [inline bootstrap/build.xml:99]: I don't see any problem here. (Note: I have rebased.)

Comment (rarguello) [inline bootstrap/build.xml:99]: I'm still seeing this problem on my repo I merged, and then fixed it again...

File: bootstrap/build.xml [hunk 2]
```diff
@@ -20,7 +20,7 @@
 
    <target name="bootstrap"
       description="Create gradle-bootstrap.jar"
-      depends="launcher,code-quality,core-impl,plugins">
+      depends="launcher,core-impl,plugins,code-quality">
```
Comment (rarguello) [inline bootstrap/build.xml:23]: Order is important here, the 'code-quality' target depends on the 'plugins' target being executed first

Comment (wolfc) [inline bootstrap/build.xml:23]: It is not the order that is important here, the 'code-quality' target must depend on 'plugins'.

**Comment by wolfc** [comment]
This stuff was based on the fedora16 branch, not on a fedora17 branch. I'm now building on F17, so I see the issues.

**Pull request closed without merging**